src/services/api/AxiosInstance.ts

```diff
@@ -73,6 +78,20 @@ const respHandler = (response: AxiosResponse) => {
   return response
 }
 
+const githubApiInterceptor = (resp: AxiosResponse) => {
+  if (
+    resp.status !== 304 &&
```

Comment:
Anything other than 304 means that a API call quota has been used right?

Reply:
yes